requirements.txt

```diff
@@ -92,7 +90,7 @@ django-typed-models==0.7.0
 django-storages==1.6.6
 google-cloud-storage==0.22.0  # dependency of django-storages, hard-pin to version
 django-dirtyfields==1.3.1
-git+https://github.com/cos-forks/django-extensions@master
+django-extensions==2.1.3
```

Comment:
@Johnetordoff Did you verify that `python manage.py osf_shell` still works as expected?